### PR TITLE
test(e2e,playwright): video-player.tsx

### PR DESCRIPTION
**Pull request by rohitrai3**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Solves "template-challenge-components/video-player" > #51705

camperbot cant code

### CODE REVIEW

File: e2e/video-player.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test.describe('Challenge Video Player Component Tests', () => {
+  test('should render video player and play button', async ({ page }) => {
+    await expect(
+      page.locator('iframe[title="YouTube video player"]')
```
Comment (naomi-lgbt) [inline e2e/video-player.spec.ts:16]: Should we add a playright test attribute to the iframe? I think that's more efficient than a `locator` query.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: e2e/video-player.spec.ts (hunk 1 above)
Comment (rohitrai3) [reply e2e/video-player.spec.ts:16]: As we are using `YouTube` component from `react-youtube` package, we don't have access to the `iframe`. One other option is to use `locator('_react=YouTube')`.
[Code](https://github.com/freeCodeCamp/freeCodeCamp/blob/225a0fbd059781fa262148e6a21e8f27d6303638/client/src/templates/Challenges/components/video-player.tsx#L2)

File: e2e/video-player.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/scientific-computing-with-python/python-for-everybody/introduction-why-program'
+  );
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
```
Comment (naomi-lgbt) [inline e2e/video-player.spec.ts:12]:
```suggestion
```

We actually don't need this bit. Playwright does this automatically.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by rohitrai3** [comment]
Removed afterEach block as suggested.
Wanted to know going forward, should i skip the similar afterEach block in tests?

**Review by naomi-lgbt — APPROVED** [review]
Yes, the afterEach can be left out if all its doing is closing the page. :3

**Review by Ahmadkashif — APPROVED** [review]
LGTM

**Pull request merged**